src/app/ApplicationTemplate.Presentation/Framework/Email/EmailAttachment.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.IO;
+
+namespace ApplicationTemplate;
+
+public sealed class EmailAttachment
+{
+	public EmailAttachment(string contentType, string fullPath)
+	{
+		if (fullPath is null)
```

Comment:
I wonder if this two checks couldn't be merged into one ? 

```csharp
if (string.IsNullOrEmpty(fullPath))
{
    throw new ArgumentException("The file path cannot be null or empty.", nameof(fullPath));
}

Reply:
Or just a `IsNullOrWhiteSpace`, unless we want to keep handling each case separately

Reply:
Fixed.

Reply:
Fixed